### DISCUSSION TITLE
Finding diffusion reaction rates.

### DISCUSSION THREAD

**Discussion by sm1216** [Open-Systems-Pharmacology/Forum#1436] [Questions & Problems]
Is there a way to find reaction-diffusion rates for the metabolites involved in the conversion of one compound to other compounds?

**Comment by StephanSchaller** [comment]
@sm1216 , can you specify what exactly you mean by "reaction-diffusion rates for the metabolites"? We here assume well-stirred conditions and no limitations of reaction rates by diffusion (i.e., no spatial distribution of concentrations within one compartment).

**Reply by sm1216** [reply]
Yes, I am asking about the metabolite's diffusion reaction rates. For how one compound changes or transforms (acetylation process) using the action of those metabolites.

like in figure 3 of the following paper.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5827733/

**Comment by Yuri05** [comment]
Is **this** what you are looking for?

https://docs.open-systems-pharmacology.org/working-with-mobi/mobi-documentation/model-building-components#reactions-and-molecules

![grafik](https://user-images.githubusercontent.com/25061876/212358878-a0e5a3e7-016a-4fc4-aad8-df96407f692a.png)
